roles/proxy/templates/nginx_artemis.conf.j2

```diff
@@ -86,7 +86,7 @@ server {
 
     location /api/authenticate {
         proxy_pass http://artemis/api/authenticate;
-        limit_req zone=loginlimit burst=5 nodelay;
+        limit_req zone=loginlimit burst=3 delay=2;
```

Comment:
can you please add a comment above this line what it actually means?

Reply:
Let me know if the comments help your understanding

Reply:
Yes, thank you